src/resources/Account.js

```diff
@@ -37,7 +37,20 @@ export default class Account extends Resource {
       user: result.user
     }
 
-    return await this.retrieve()
+    const user = await this.retrieve()
```

Comment:
Do we need this variable assignment? Can we keep the `return` statement and place it after the backoffice login?